packages/hooks-react/src/useBootstrapApp.ts

```diff
@@ -19,11 +20,13 @@ export type OnReadyCallback = (config: Config | undefined) => void;
 
 export const useBootstrapApp = (url: string, onReady: OnReadyCallback) => {
   const queryClient = useQueryClient();
+  const { i18n } = useTranslation('menu');
+
   const refreshEntitlements = () => queryClient.invalidateQueries({ queryKey: ['entitlements'] });
 
   const { data, isLoading, error, isSuccess, refetch } = useQuery<Resources, Error | AppError>(
     'config-init',
-    () => applicationController.initializeApp(url, refreshEntitlements),
+    () => applicationController.initializeApp(url, i18n.language, refreshEntitlements),
```

Comment:
Hm, I think that here we can simply import i18n library and get language from there. This one doesn't need to be dynamic.